pkg/component/controller/etcd.go

```diff
@@ -291,7 +291,8 @@ func (e *Etcd) Healthy() error {
 }
 
 func detectUnsupportedEtcdArch() error {
-	if strings.Contains(runtime.GOARCH, "arm") {
+	// https://github.com/etcd-io/etcd/blob/v3.5.2/server/etcdmain/etcd.go#L467-L472
+	if runtime.GOARCH != "amd64" || runtime.GOARCH == "arm64" {
```

Comment:
Is this logic correct? I would expect:
```
if runtime.GOARCH != "amd64" && runtime.GOARCH != "arm64" {
```

Alternative:
```
if runtime.GOARCH == "amd64" || runtime.GOARCH == "arm64" {
    return nil
}
```

Reply:
🤦‍♂️ if you decide to do it the other way round after having done the first half already. Thanks for spotting!

Reply:
Fixed and thanks again!